scripts/bootstrap

```diff
@@ -251,6 +251,9 @@ blockchain_configuration:
       ratio: "1/1"
       epoch_start: 0
       epoch_rate: 1
+      rewards_limit: "15/100"
+      npools: 1000
+      npools_threshold: 300
```

Comment:
Don't set them in the bootstrap. You should leave the default (no limits, no threashold, no max).

Reply:
These must be set in the bootstrap since there is no way to change them later in the testnet.  They are key parameters to ensure that the system behaves as intended.

I will discuss the actual values to make sure they have the intended effect.  npools = 1000 may be too high and the threshold might be set higher.

Reply:
bootstrap is not the script used to setup the IOHK's incentivised testnet. So no need to add that.

Reply:
But the parameters will also be fixed for the testnet, and we can expect users to pick this up to run their own equivalents for test purposes.  As I said, providing other examples is good, but you do want something that's consistent with the actual testnet settings.

Reply:
there is no need to be consistent, this script is meant to help bootstrapping a simple node and start doing some development tests.

Reply:
OK.  Have created an additional example (and tested our parameters using it).